test/integration/authorization_test.rb

```diff
@@ -34,7 +34,7 @@ class LoginTest < ActionDispatch::IntegrationTest
   test 'you can logout' do
     user = create(:user, :default)
     visit root_path
-    fill_in 'session[email_address]', with: 'richard@turing.io'
+    fill_in 'session[username]', with: 'richard'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.